checker/types/mapping.go

```diff
@@ -12,36 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package checker
+package types
 
 import (
 	"fmt"
-	"github.com/google/cel-go/semantics/types"
+	"github.com/google/cel-spec/proto/checked/v1/checked"
```

Comment:
Suggest using `pb` as short name for proto package.

```go
	pb "github.com/google/cel-spec/proto/checked/v1/checked"
```

Reply:
Deferring to another PR.